src/hearings/containers/request-hearing/hearing-link/hearing-link.component.ts

```diff
@@ -3,9 +3,11 @@ import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { ActivatedRoute, Router } from '@angular/router';
 import { select, Store } from '@ngrx/store';
 import { Subscription } from 'rxjs';
+import { switchMap } from 'rxjs/operators';
+import { HearingLinksStateData } from 'src/hearings/models/hearingLinksStateData.model';
```

Comment:
@johnbenjamin-hmcts - please change this import to a relative path

Reply:
I am not sure how it sneaks in like spyware even after being extra cautious :(

Thank you @Daniel-Lam I have now changed the path.